tests/test/fn_def.rs

```diff
@@ -0,0 +1,66 @@
+use super::*;
```

Comment:
Another boo-boo, these tests aren't run because the module is not listed in `tests/test/mod.rs`

Reply:
They fail in the parser because it expects `;` instead of `{ }`.

Reply:
🙄